superset/db_engine_specs.py

```diff
@@ -590,6 +590,58 @@ def get_table_names(cls, inspector, schema):
         """Need to disregard the schema for Sqlite"""
         return sorted(inspector.get_table_names())
 
+class DrillEngineSpec(BaseEngineSpec):
+    """Engine spec for Apache Drill"""
+    engine = 'drill'
+
+    @classmethod
+    def extra_table_metadata(cls, database, table_name, schema_name):
+        """Returns engine-specific table metadata"""
+
+        return {}
+
+    @classmethod
+    def select_star(cls, my_db, table_name, engine, schema=None, limit=100,
+                    show_cols=False, indent=True, latest_partition=True,
+                    cols=None):
+        table_name = table_name.replace('@@@', '.')
```

Comment:
A comment would be nice here, and not hard coding would be nice too. You could use a constant in module scope or better, import the constant from the related dialect package. Something like `from sqlalchemy_drill import DRILL_SCHEMA_TABLE_DELIMITER` or whatever it would be declared as on the other side.

Wherever that constant is declared or used, it could be good to reference documentation that explains the hack and why it's necessary.

Reply:
Fixed.